src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objStyles = {};
+
+  sourceString.split(';').map((style) => {
+    const arrStyle = style.split(':').map((el) => el.trim());
+
+    objStyles[arrStyle[0]] = arrStyle[1];
+  });
```

Comment:
The `map` method is being used incorrectly. It should be used when you want to transform each element of an array and return a new array, not for side effects on an object. Consider using `forEach` or `reduce` for this purpose.